src/app/modules/ix-table2/interfaces/data-provider.interface.ts

```diff
@@ -2,12 +2,15 @@ import { Observable } from 'rxjs';
 import { TablePagination } from 'app/modules/ix-table2/interfaces/table-pagination.interface';
 import { TableSort } from 'app/modules/ix-table2/interfaces/table-sort.interface';
 
-export interface TableProvider<T> {
+export interface DataProvider<T> {
   rows: T[];
```

Comment:
`rows` should be removed as it now means different things depending on the child class used.

Reply:
done!